src/Screens/Home/Home.css

```diff
@@ -0,0 +1,56 @@
+.main_home {
+  width: 100%;
+}
+
+.main_home {
```

Comment:
Element is targeted twice

Reply:
Thanks for pointing, resolved in https://github.com/saadahmadhere/Stopphur/pull/3/commits/974e2723d8d7d3d3efab0c6f1723715ab05a0fa7